README.md

```diff
@@ -5,21 +5,15 @@ examples:
 
 RP2040:
 
-```
-python3 parser.py "Adafruit Feather RP2040.fzpz" C:\Users\ladyada\Dropbox\micropython\circuitpython\ports\raspberrypi\boards\adafruit_feather_rp2040\pins.c rp2040pins.csv
-```
+```python3 parser.py "Adafruit Feather RP2040.fzpz" ~/Git/circuitpython/ports/raspberrypi/boards/adafruit_feather_rp2040/pins.c rp2040pins.csv```
 
-```
-python3 parser.py "Adafruit ItsyBitsy RP2040.fzpz" C:\Users\ladyada\Dropbox\micropython\circuitpython\ports\raspberrypi\boards\adafruit_feather_rp2040\pins.c rp2040pins.csv
-```
+```python3 parser.py "Adafruit ItsyBitsy RP2040.fzpz" ~/Git/circuitpython/ports/raspberrypi/boards/adafruit_feather_rp2040/pins.c rp2040pins.csv```
```

Comment:
Same suggestion as above. Also this path needs `feather` to be updated to `itsybitsy`.
```suggestion
```python3 parser.py "Adafruit ItsyBitsy RP2040.fzpz" _~/Git/circuitpython/ports/raspberrypi/boards/adafruit_itsybitsy_rp2040/pins.c rp2040pins.csv```
```

Reply:
Fixed